diff --git a/api/gmsh.py b/api/gmsh.py
index 02f124062..dcf4f97d5 100644
--- a/api/gmsh.py
+++ b/api/gmsh.py
@@ -14,19 +14,20 @@
 from ctypes import *
 import signal
 import os
+import sys
 import platform
 from math import pi
 
 GMSH_API_VERSION = "4.1"
 
 signal.signal(signal.SIGINT, signal.SIG_DFL)
-libdir = os.path.dirname(os.path.realpath(__file__))
+libdir = sys.prefix
 if platform.system() == 'Windows':
-    lib = CDLL(os.path.join(libdir, "gmsh-4.1.dll"))
+    lib = CDLL(os.path.join(libdir, "Library", "bin", "gmsh-{}.dll".format(GMSH_API_VERSION)))
 elif platform.system() == 'Darwin':
-    lib = CDLL(os.path.join(libdir, "libgmsh.dylib"))
+    lib = CDLL(os.path.join(libdir, "lib", "libgmsh.dylib"))
 else:
-    lib = CDLL(os.path.join(libdir, "libgmsh.so"))
+    lib = CDLL(os.path.join(libdir, "lib", "libgmsh.so"))
 
 use_numpy = False
 try:
