diff --git a/tutorial/c++/t13.cpp b/tutorial/c++/t13.cpp
index caf9cc105..5acc8026c 100644
--- a/tutorial/c++/t13.cpp
+++ b/tutorial/c++/t13.cpp
@@ -123,7 +123,7 @@ int main(int argc, char **argv)
   auto checkForEvent = [=]() -> bool {
     std::vector<std::string> action;
     gmsh::onelab::getString("ONELAB/Action", action);
-    if(action.size() and action[0] == "check") {
+    if(action.size() && action[0] == "check") {
       gmsh::onelab::setString("ONELAB/Action", {""});
       createGeometryAndMesh();
       gmsh::graphics::draw();
diff --git a/tutorial/c++/t21.cpp b/tutorial/c++/t21.cpp
index 5891a7ede..e6fbbf096 100644
--- a/tutorial/c++/t21.cpp
+++ b/tutorial/c++/t21.cpp
@@ -188,7 +188,7 @@ int main(int argc, char **argv)
   auto checkForEvent = [=]() -> bool {
     std::vector<std::string> action;
     gmsh::onelab::getString("ONELAB/Action", action);
-    if(action.size() and action[0] == "check") {
+    if(action.size() && action[0] == "check") {
       gmsh::onelab::setString("ONELAB/Action", {""});
       partitionMesh();
       gmsh::graphics::draw();
diff --git a/tutorial/c++/t3.cpp b/tutorial/c++/t3.cpp
index cc2490557..1b7b30da4 100644
--- a/tutorial/c++/t3.cpp
+++ b/tutorial/c++/t3.cpp
@@ -125,7 +125,7 @@ int main(int argc, char **argv)
   auto checkForEvent = [=]() -> bool {
     std::vector<std::string> action;
     gmsh::onelab::getString("ONELAB/Action", action);
-    if(action.size() and action[0] == "check") {
+    if(action.size() && action[0] == "check") {
       gmsh::onelab::setString("ONELAB/Action", {""});
       createGeometryAndMesh();
       gmsh::graphics::draw();
