diff --git a/src/common/gmsh.cpp b/src/common/gmsh.cpp
index 31dc8961e..907d79447 100644
--- a/src/common/gmsh.cpp
+++ b/src/common/gmsh.cpp
@@ -5,6 +5,13 @@
 
 #include <sstream>
 #include "GmshConfig.h"
+#if defined(HAVE_FLTK) && defined(_WIN32)
+#define NOMINMAX
+// Opencascade define NOMSG and a bunch of other macros to avoid some
+// definitions from windows.h, but FLTK needs MSG macro.
+// Including windows.h at the beginning makes defining NOMSG irrelevant.
+#include "windows.h"
+#endif
 #include "GmshDefines.h"
 #include "GmshGlobal.h"
 #include "MallocUtils.h"
