diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4122a22..1d29390 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1334,12 +1334,12 @@ if(ENABLE_OCC)
     list(APPEND EXTERNAL_INCLUDES ${OCC_INC})
     if(HAVE_64BIT_SIZE_T)
       add_definitions(-D_OCC64)
-    endif()
-    if(WIN32)
+    endif()    
+    if(MSVC)
+      add_definitions(-DWNT)
+    elseif(WIN32)
       list(APPEND EXTERNAL_LIBRARIES "winspool")
       add_definitions(-DOCC_CONVERT_SIGNALS)
-    elseif(MSVC)
-      add_definitions(-DWNT)
     endif()
   endif()
 endif()
